lib/src/features/devices/data/repository/timer_repository_impl.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:zone_app/src/core/failures.dart';
+import 'package:dartz/dartz.dart';
+import 'package:zone_app/src/features/devices/domain/repository/timer_repository.dart';
+
+class TimerRepositoryImpl implements TimerRepository {
+  @override
+  Either<Failure, Stream<int>> createTimer({required int ticks}) {
+    Stream<int> timer =
+        Stream.periodic(const Duration(seconds: 1), (x) => ticks - x - 1)
```

Comment:
Put the number in a Variable. Because I don't know what is 1
It could be (?)
```suggestion
        Stream.periodic(const Duration(seconds: FIRST_SECOND), (x) => ticks - x - FIRST_SECOND)
```

Reply:
I don't see it as useful. The stream is running the internal formula each second